libraries/AP_Notify/SITL_SFML_LED.cpp

```diff
@@ -0,0 +1,83 @@
+/*
+   Copyright (C) 2019 Peter Barker. All rights reserved.
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include <AP_HAL/AP_HAL.h>
+
+#include <stdio.h>
+#include <unistd.h>
+
+#ifdef WITH_SITL_RGBLED
```

Comment:
Nitpicking: putting this first prevents recompilation of this file if something changes in AP_HAL.h (or the C headers).